plugins/nsenter.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: nsenter
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/pabateman/kubectl-nsenter
+  shortDescription: "Execute shell command under container namespace of pod"
+  description: |
+    Kubectl plugin for pod's linux namespaces command execution via direct node ssh connection
```

Comment:
This is not very descriptive, can you rephrase this like:

> This plugin establishes a connection to node that's running the Pod over SSH and uses
> nsenter to run commands in the container's namespace. You would need something
> like this to run programs that are missing in the container image.

not verbatim but your own summary.